src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -284,6 +284,17 @@
       <LatestCommit>N/A</LatestCommit>
     </PropertyGroup>
 
+
+    <!-- If things match expected patterns, create $(GitBranchName) from $(RawGitBranchName) -->
+    <Message Text="GIT Branch Name as seen by git describe: '$(RawGitBranchName)'" /> 
+    <PropertyGroup Condition="$(RawGitBranchName.StartsWith('he:ads/'))">
```

Comment:
Is the `:` in `'he:ads/'` a typo?

Reply:
For what it was worth, the first two issues were because I staged the edit early and forgot to stage it again (thus pushed stale things).  Anyway it is fixed now.